tests/test_report_manager.py

```diff
@@ -295,10 +307,10 @@ def subtest_build_summary_table(self, mode, cpu_only):
             )
 
     def test_build_detailed_info(self):
-        for cpu_only in [True, False]:
-            self._subtest_build_detailed_info(cpu_only)
+        for report_gpu_metrics in [True, False]:
+            self._subtest_build_detailed_info(report_gpu_metrics)
```

Comment:
Is there a reason why we don't loop over `cpu_only` options here?

Reply:
Yes. Because I forgot to add it! This is fixed now.